setup.py

```diff
@@ -112,8 +112,14 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'matplotlib', 'scipy', 'poppy', 'astropy'],
-      setup_requires=['numpy', 'astropy'],
+      install_requires=[
+          'numpy>=1.10.0',
+          'matplotlib>=1.5.3',
```

Comment:
Why are you pushing these versions to the most recent? I predict there are a LOT of users out there who don't have the most recent matplotlib that was released just a few weeks ago. I know it fixed that bug you were talking about, but is it strictly necessary as an "install requires"?